### PR TITLE
docs: :memo: decision post for versioning with Commitizen

### LINKED ISSUE #107: Decision post on why semantic release

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

Closes #107

## Reviewer Focus

This PR needs a fairly short review.

## Checklist

- [x] Ran spell-check
- [x] Formatted Markdown
- [ ] Rendered website locally

### CODE REVIEW

File: why-semantic-release-with-commitizen/index.qmd [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+::: content-hidden
+State the context and some background on the issue, then write a
+statement in the form of a question for the problem.
+:::
+
+Setting and updating the version of software communicates to users the
+state and changes of software. However, having to manually update the
+version as well as deciding when a given state of the software requires
+a version or not is time- and energy-consuming. So instead, we would
```
Comment (martonvago) [inline why-semantic-release-with-commitizen/index.qmd:25]:
```suggestion
a version is time- and energy-consuming. So instead, we would
```
Or change "when" to "whether"?

File: why-semantic-release-with-commitizen/index.qmd [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    book).
+-   Very actively maintained and developed.
+-   Can also generate an auto-generated changelog.
+:::
+
+::: column
+#### Drawbacks
+
+-   The website is difficult to navigate, relative to Commitizen at
+    least. For instance, there isn't any tutorials or "getting started"
```
Comment (martonvago) [inline why-semantic-release-with-commitizen/index.qmd:93]:
```suggestion
    least. For instance, there aren't any tutorials or "getting started"
```

**Review by martonvago — CHANGES_REQUESTED** [review]
Very nice!!

**Review by K-Beicher — APPROVED** [review]
Looks good to me

**Pull request merged**